specification/mysql/resource-manager/Microsoft.DBforMySQL/stable/2017-12-01/mysql.json

```diff
@@ -1595,8 +1934,41 @@
       },
       "description": "Billing information related properties of a server."
     },
+    "ResourceIdentity": {
+      "description": "Azure Active Directory identity configuration for a resource.",
+      "type": "object",
+      "properties": {
+        "principalId": {
+          "format": "uuid",
+          "description": "The Azure Active Directory principal id.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The identity type. Set this to 'SystemAssigned' in order to automatically create and assign an Azure Active Directory principal for the resource.",
+          "enum": [
+            "SystemAssigned"
```

Comment:
This looks old -does this only allow SystemAssigned identities, or are user-assigned identities allowed as well?

Reply:
No we don't support user assigned and we don't have that in  our road map